examples/CameraPositionerWidget.html

```diff
@@ -60,30 +43,23 @@
           extent
         );
 
-        udvizBrowser.addElevationLayer(
-          configs['elevation'],
-          frame3DPlanar.itownsView,
-          extent
-        );
-
-        udvizBrowser.addGeoJsonLayers(
-          configs['geoJSONs'],
-          frame3DPlanar.itownsView,
-          extent
+        // //// CAMERA POSITIONER
+        const cameraPosition = new udvizBrowser.Widget.CameraPositioner(
+          frame3DPlanar.getItownsView()
         );
 
-        // //// ABOUT MODULE
-        const about = new udvizBrowser.Widget.AboutWindow(configs['about']);
-        app.addWidgetView('about', about);
+        const parent = document.createElement('div');
+        parent.style.backgroundColor = 'white';
```

Comment:
Since this is an example imo [there are no good or bad practices](https://www.leparisien.fr/resizer/waI87oyiJ3zfhJsMWl6zbUOfVRE=/1280x634/arc-anglerfish-eu-central-1-prod-leparisien.s3.amazonaws.com/public/5L6V5XJLBINOHWNPBYVSRE5LRY.jpg) as is up to the user to do what he wants (this is not an ud-viz problem). But you right this line is bad...

We could add in default.css a class parentFitContent since all parent element created in examples are quite the same, but I think the problem here is that the widget html do not render properly when it's not encapsulated in parent element with width:fit-content. We could then consider that this css should be bring to the widget html.